src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.targets

```diff
@@ -166,6 +175,15 @@ Copyright (C) Microsoft Corporation. All rights reserved.
           Condition=" '$(DeployOnBuild)' == 'true' " >
   </Target>
 
+  <Target Name="CheckContainersPackage" AfterTargets="Build" Condition="'$(EnableSdkContainerSupport)' == 'true'">
+    <PropertyGroup>
+      <ContainersPackageIdentity>Microsoft.NET.Build.Containers</ContainersPackageIdentity>
+    </PropertyGroup>
+    <ItemGroup>
+      <ContainersPackage Include="@(PackageReference)" Condition="'%(Identity)' == '$(ContainersPackageIdentity)'"/>
+    </ItemGroup>
+    <Warning Text="Microsoft.NET.Build.Containers package is explicitly referenced. Consider removing the package reference to Microsoft.NET.Build.Containers as it is now part of .NET SDK." Condition="'@(ContainersPackage)' != ''" />
```

Comment:
Should this error message say NuGet package to be clear so that it won't be confused with any other kind of package.